crates/bins/node/src/main.rs

```diff
@@ -1,14 +1,18 @@
-#[cfg(not(any(feature = "starknet")))]
+#[cfg(all(feature = "mock", feature = "starknet"))]
+compile_error!("Only one of the features 'mock' and 'starknet' can be enabled at the same time");
+#[cfg(not(any(feature = "mock", feature = "starknet")))]
```

Comment:
things have changed here.

You messed up during rebase I believe.

we wan t`#[cfg(not(any(feature = "starknet")))]` as it is on main

Reply:
this is why your CI fail